src/app/webbrowser/BrowserTab.qml

```diff
@@ -44,25 +44,19 @@ FocusScope {
     readonly property real lastCurrent: internal.lastCurrent
     property bool incognito
     readonly property bool empty: !url.toString() && !initialUrl.toString() && !restoreState && !request
+    property bool loadingPreview: false
+    readonly property size previewSize: Qt.size(webview.width*Screen.devicePixelRatio,
+                                                webview.height*Screen.devicePixelRatio)
+    readonly property size previewThumbnailSize: Qt.size(webview.width/1.5,
+                                                         webview.height/1.5)
```

Comment:
The following warnings occur upon closing a tab:

```
file:///usr/share/morph-browser/webbrowser/BrowserTab.qml:48: TypeError: Cannot read property 'width' of null
file:///usr/share/morph-browser/webbrowser/BrowserTab.qml:50: TypeError: Cannot read property 'width' of null
file:///usr/share/morph-browser/webbrowser/BrowserTab.qml:48: TypeError: Cannot read property 'width' of null
file:///usr/share/morph-browser/webbrowser/BrowserTab.qml:50: TypeError: Cannot read property 'width' of null
```

I know the tab is closing anyway and these warnings probably don't matter, but I prefer not adding new ones.

Reply:
Fixed.